ConsentViewController/Classes/SourcePointClient/MessagesRequest.swift

```diff
@@ -27,9 +27,15 @@ struct MessagesRequest: QueryParamEncodable {
                 let idfaSstatus: SPIDFAStatus
             }
 
+            // TODO: find out if there are other parameters needed
```

Comment:
Be aware of this comment, pls remove if not important

Reply:
Thanks. Confirming now with Sid and Dan if we have the right variables.